src/transaction_utils.cpp

```diff
@@ -421,7 +421,8 @@ namespace sdk {
 
         // Transactions with outputs below the dust threshold (except OP_RETURN)
         // are not relayed by network nodes
-        if (!result.value("send_all", false) && satoshi.value() < session.get_dust_threshold()) {
```

Comment:
needs a comment here that we ignore this check for greedy outputs until later when their amount is populated.

Reply:
Comment added, thanks